Misc/NEWS.d/next/Library/2019-05-30-13-02-31.bpo-37101.pJ8FSq.rst

```diff
@@ -0,0 +1 @@
+Modified logging.Filterer.filter() to use efficient, readable built-ins.
```

Comment:
I'd suggest something more along the lines of

```
Improved the efficiency of :mod:`logging` record filtering.
```

Reply:
Do I just edit the file directly? Or use the `blurb` tool somehow?

Reply:
Edit directly.

Reply:
Done. :smile: